.github/workflows/test-browser.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         if: ${{ matrix.chrome }}
         run: |
           chrome --version
-          echo CHROME_BIN="$(which chrome)" | tee -a "$GITHUB_ENV"
+          { echo CHROME_BIN="$(which chrome)"; echo CHROME_TEST=1; } | tee -a "$GITHUB_ENV"
```

Comment:
In the final iteration, I was using Karma's `CHROME_BIN` var as the "test this browser" flag. But I wasn't aware that that var is automatically set to the pre-installed Chrome version on the GitHub Actions machine. So I added a custom var, `CHROME_TEST` (and `FIREFOX_TEST`). Fixed.